tests/e2e/vsphere_volume_expansion.go

```diff
@@ -730,19 +736,18 @@ var _ = ginkgo.Describe("Volume Expansion Test", func() {
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		gomega.Expect(pvclaim).NotTo(gomega.BeNil())
 
-		if !featureEnabled {
-			ginkgo.By("File system resize should not succeed since SPS service is down" +
-				" and cns new sync feature is disabled. Expecting an error")
-			expectedErrMsg = "VolumeResizeFailed"
-			framework.Logf("Expected failure message: %+q", expectedErrMsg)
-			isFailureFound, err := getEventsListAndVerifyPvcError(client, namespace, pvclaim, expectedErrMsg)
-			gomega.Expect(err).NotTo(gomega.HaveOccurred())
-			gomega.Expect(isFailureFound).To(gomega.BeTrue(), "Expected error %v, to occur but did not occur", expectedErrMsg)
-			ginkgo.By("Bringup SPS service")
-			err = invokeVCenterServiceControl(startOperation, spsServiceName, vcAddress)
-			gomega.Expect(err).NotTo(gomega.HaveOccurred())
-			isSPSServiceStopped = false
-		}
+		ginkgo.By("File system resize should not succeed since SPS service is down. " +
+			"Expecting an error")
+		expectedErrMsg = "VolumeResizeFailed"
+		framework.Logf("Expected failure message: %+q", expectedErrMsg)
+		isFailureFound, err := getEventsListAndVerifyPvcError(client, namespace, pvclaim, expectedErrMsg)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		gomega.Expect(isFailureFound).To(gomega.BeTrue(), "Expected error %v, to occur but did not occur", expectedErrMsg)
+
+		ginkgo.By("Bringup SPS service")
+		err = invokeVCenterServiceControl(startOperation, spsServiceName, vcAddress)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
```

Comment:
Please add this to check the SPS Runing condition  in all the places where ever we are starting the service
err = waitVCenterServiceToBeInState(serviceName, vcAddress, svcRunningMessage)
gomega.Expect(err).NotTo(gomega.HaveOccurred())